server/src/test/java/com/paypal/selion/utils/ConfigParserTest.java

```diff
@@ -15,26 +15,17 @@
 
 package com.paypal.selion.utils;
 
-import static org.testng.Assert.*;
-import static org.powermock.api.mockito.PowerMockito.*;
+import static org.testng.Assert.assertEquals;
 
 import java.io.File;
 
-import org.mockito.Mockito;
-import org.powermock.core.classloader.annotations.PrepareForTest;
-import org.powermock.modules.testng.PowerMockTestCase;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
-@PrepareForTest(ConfigParser.class)
```

Comment:
It should, But I didn't face the issue. Let me check once again and update the code if required.